src/index.js

```diff
@@ -1,7 +1,7 @@
 import { getOptions } from "loader-utils";
 import { validate } from "schema-utils";
 
-import postcss from "postcss";
+import ourPostcss from "postcss";
```

Comment:
Let's return to `postcss`

Reply:
The problem I ran into was naming the local variable `postcss` too. I can change that and put this back to `postcss`